frontend/views/top_containers/bulk_operations/_bulk_action_templates.html.erb

```diff
@@ -4,4 +4,90 @@
+          <div class="controls">
+            <%= text_field_tag "ils_holding_id", params["ils_holding_id"], :style => "width: 80%;", :disabled => "disabled", :id => "ils_holding_id" %>
+            <%= check_box_tag "update_ils_holding_id", params["update_ils_holding_id"], false, :onclick => "handleClick(this);", :name => "fieldToggle" %>
+          </div>
+        </div>
+        <div class="control-group">
+          <label class="control-label" for="ils_item_id">ILS Item ID</label>
+          <div class="controls">
+            <%= text_field_tag "ils_item_id", params["ils_item_id"], :style => "width: 80%;", :disabled => "disabled", :id => "ils_item_id" %>
+            <%= check_box_tag "update_ils_item_id", params["update_ils_item_id"], false, :onclick => "handleClick(this)", :name => "fieldToggle" %>
+          </div>
+        </div>
+        <div class="control-group">
+          <div class="controls">
+            <%= submit_tag "Update ${selection.length} records", :id => "bulk_update_submit", :class => "btn btn-primary", :disabled => "disabled" %>
+          </div>
+        </div>
+      </div>
+    <% end %>
+
+    <script>
+      $('#bulk_update_form').on("submit", function(event) {
+        event.preventDefault();
+        var ids = '';
+        {for container in selection}
+          ids += "${container[0]}".replace(/(.+\/)/, '') + ',';
+        {/for}
+        ids = ids.slice(0,-1);
+        data = $('#bulk_update_form').serializeArray();
+        data.push({name: "ids", value: ids});
+        $.ajax({
+          url:"/plugins/top_containers/bulk_operations/update",
+          data: data,
+          type: "post",
+          success: function(html) {
+            $.each($(":input[name=fieldToggle]"), function(idx, cb) {
+              if (cb.checked) {
+                cb.click();
+              }
+            });
+            $('#alertBucket').replaceWith(html);
+          },
+          error: function(jqXHR, textStatus, errorThrown) {
+            $('#alertBucket').replaceWith('<div id="alertBucket" class="alert alert-error">' + jqXHR.responseText + '</div>');
+          }
+        });
+      });
+
+      function formChanged() {
+        var anyon = false;
+        $.each($(":input[name=fieldToggle]"), function(idx, cb) {
+          if (cb.checked) {
+            anyon = true;
+          }
+        });
+        if (anyon) {
+          $('#alertBucket').replaceWith('<div id="alertBucket" class="alert alert-warning">You are about to update multiple records.</div>');
+          $('#bulk_update_submit').removeAttr('disabled')
+        } else {
+          $('#alertBucket').replaceWith('<div id="alertBucket" class="alert alert-info">Click the checkbox of the fields to update.</div>');
+          $('#bulk_update_submit').attr('disabled', 'disabled')
+
+        }
+      }
+
+
+      function handleClick(cb) {
+        id = cb.id.replace(/[^_]+_/, '');
+        elt = $('#' + id);
```

Comment:
As we generate both the checkbox and input at the same time, could we store the input's id on the checkbox as a data attribute? It would save us having to parse the id string.